spec/controllers/laa_references_controller_spec.rb

```diff
@@ -73,19 +74,24 @@
       }
     end
 
+    it 'returns unauthorized' do
+      put :record_representation_order, params: laa_reference_params
+      expect(response).to be_unauthorized
+    end
+
     context 'when the LaaReference exists' do
-      let(:laa_reference_id) { laa_reference.id }
+      before { laa_reference }
```

Comment:
Should this be `laa_reference.save!` as it is in the other specs?

Reply:
Yep, that's better, good catch :+1: